ethereum/contracts/ETHApp.sol

```diff
@@ -1,32 +1,45 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.6.2;
+pragma experimental ABIEncoderV2;
```

Comment:
@vgeddes actually this is a mission because our dependencies (openzeppelin) dont support 0.8.0 on master yet and truffle doesn't easily allow mixing versions.